test/functional/page_objects/settings_page.ts

```diff
@@ -872,8 +875,7 @@ export class SettingsPageObject extends FtrService {
 
   async setFieldType(type: string) {
     this.log.debug('set type = ' + type);
-    await this.testSubjects.setValue('typeField', type);
-    await this.browser.pressKeys(this.browser.keys.ENTER);
+    await this.comboBox.set('typeField', type);
```

Comment:
We could also try to wait here for the value to actually change instead of the `setTimeout` later.